contracts/NFTStakingManager.sol

```diff
@@ -38,7 +38,17 @@ struct EpochInfo {
   uint256 totalStakedLicenses;
 }
 
+enum MyStatus {
```

Comment:
This is not a great name? Also, not sure why we need it? If we set `startEpoch` during `completeValidatorRegistration` then thats all we need right?

Reply:
you're right, just experimenting. deleted validator status entirely